run/filesystem/index.js

```diff
@@ -0,0 +1,97 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+/**
+ * Express webapp that generates files to a mounted NFS  when deployed to Cloud Run.
+ *
+ * See https://cloud.google.com/run/docs/tutorials/network-filesystems-filestore before running the code snippet.
+ */
+
+const express = require('express');
+const rateLimit = require('express-rate-limit');
+const fs = require('fs');
+const app = express();
+const mntDir = process.env.MNT_DIR || '/mnt/nfs/filestore';
+const port = parseInt(process.env.PORT) || 8080;
+const limit = rateLimit({
+  windowMs: 15 * 60 * 1000,
+  max: 100,
+  message: 'Rate limit exceeded',
+  headers: true,
+});
+
+app.use(limit);
+app.use(mntDir, express.static(mntDir));
+
+const server = app.listen(port, () => {
+  console.log(`Listening on port ${port}`);
+});
+
+app.get(mntDir, async (req, res) => {
```

Comment:
This doesn't add the "back" button to get back to the mount directory index.

Reply:
This sample can't add the back button because it's returning the actual file which is a plain-text .txt without HTML

Reply:
The canonical does this differently. Do you have a reason to break from the canonical?